pallets/did/src/benchmarking.rs

```diff
@@ -26,6 +26,7 @@ use sp_runtime::traits::Hash;
 
 const SEED: u32 = 0;
 
+//TODO: Need to update this to match the latest version of the pallet.
```

Comment:
That's also related to the pipeline: https://github.com/KILTprotocol/mashnet-node/pull/127/checks?check_run_id=2200920675

Reply:
> since it's not compiling, might be better to remove it. Do we have a ticket for adding benchmarking?

Now we do: [Ticket 1197](https://github.com/KILTprotocol/ticket/issues/1197)